### PR TITLE
Add arm64/aarch64 architecture support

### PR DESCRIPTION
**Pull request by CheyiLin**

### CODE REVIEW

File: bin/install [hunk 1]
```diff
@@ -8,15 +8,30 @@ function install() {
...
+  local platform="$(get_os)_$(get_arch)"
+  local download_url="https://github.com/fullstorydev/grpcurl/releases/download/v${version}/grpcurl_${version}_${platform}.tar.gz"
 
   echo "Downloading grpcurl from ${download_url}"
 
   mkdir -p "$bin_install_path"
   curl -sL "$download_url" | tar xz -C "$bin_install_path" "$file"
 }
 
+get_os() {
```
Comment (superbrothers) [inline bin/install:20]:
```suggestion
function get_os() {
```

File: bin/install [hunk 2]
```diff
@@ -8,15 +8,30 @@ function install() {
...
+  local os=$(uname -s | tr '[:upper:]' '[:lower:]')
+  case $os in
+    darwin) os="osx" ;;
+  esac
+  echo "$os"
+}
+
+get_arch() {
```
Comment (superbrothers) [inline bin/install:28]:
```suggestion
function get_arch() {
```

File: bin/install (hunk 1 above)
Comment (CheyiLin) [reply bin/install]: Fixed.

File: bin/install (hunk 2 above)
Comment (CheyiLin) [reply bin/install]: Fixed.

**Comment by superbrothers** [comment]
Thanks for your contribution.

**Pull request merged**